web_widget_options/static/src/js/web_widgets.js

```diff
@@ -0,0 +1,73 @@
+openerp.web_widget_options = function(instance, m) {
+  var _t = instance.web._t,
+      QWeb = instance.web.qweb;
+  var wwo = instance.wwo = {};
+  var o2m = instance.wwo.o2m = {}
+
+  o2m.rules = {
+    groups: function (rule, context) {
+      var user_id = instance.session.uid,
+          users = new instance.web.Model('res.users'),
+          query = users.query(['name', 'groups_id']).filter([['id', '=', user_id]]);
+
+      return query.all().then(function (res) {
+               var user = res[0];
+
+               return rule.reduce(function (accum, val) {
```

Comment:
Something like this? 

    return _.intersection(rule, user.groups_id).length = rule.length